charts/service-binding-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: service-binding-controller-manager
+  name: service-binding-operator
+  namespace: service-binding-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: service-binding-controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        control-plane: service-binding-controller-manager
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      containers:
+      - args:
+        - --leader-elect
+        - --zap-encoder=json
+        - --zap-log-level=info
+        command:
+        - /manager
+        image: quay.io/redhat-developer/servicebinding-operator:31151ab8
```

Comment:
is this the published 1.0.1 version? It should referred via sha, and there should be an option to pass another value from values.yaml

Reply:
Yes this is the published 1.0.1 version image. 
The ability to change the image tag is not provided because we will be rolling out different charts as and when there is a change in app version of sbo itself. So that way the need is not there.

Reply:
well then, please refer it via sha, not via tag. Also, enabling user to refer an alternative image is always super userful, due to various reasons - for example user is not able to access quay directly, so he/she needs to copy the image to some other internal registry, etc.

Reply:
good usecase @pedjak, so, maybe we could provide the registry as optional for internal network scenario, but the image name and sha should be hardcoded since it belongs to the app version we are putting in the Chart.yaml

Reply:
@Kartikey-star ^^^^

Reply:
There should be a good default, and this comes into `values.yaml`, not here - so that users can override it.

Reply:
Yes will do so.Will refer the image tag with the sha and image.repositpry will be a key in values.yaml with default as quay.io/redhat-developer.